composer.json

```
@@ -17,7 +17,7 @@
     }
   },
   "require" : {
-    "php": "^8"
+    "php": ">=8.0 <8.3"
```

```suggestion
    "php": "^8.0 || ^8.1 || ^8.2"
```
Explicit is better than implicit.